configs/build.tslint.json

```diff
@@ -1,7 +1,8 @@
-// Lint rules to use when building.  We don't include warnings to avoid
-// polluting the build output.
 {
+  "//": [
+    "Lint rules to use when building. We don't include warnings to avoid polluting the build output."
```

Comment:
Just wondering what's the justification for this, some tool doesn't like the comments (which don't exist in plain json)?

Reply:
That's why I moved it as a property, it was an error in my editor.

Reply:
Ok!